src/segments/battery.go

```diff
@@ -32,6 +32,10 @@ func (b *Battery) Template() string {
 }
 
 func (b *Battery) Enabled() bool {
+	if !b.env.IsWsl2() {
```

Comment:
Don't we need to validate for WSL first? As this could ruin standard linux/macos functionality?

Reply:
isWsl2 implicitly calls isWsl first